src/core/EdFi.DataManagementService.Core.External/Backend/DeleteResult.cs

```diff
@@ -28,8 +28,7 @@ public record DeleteFailureReference(string ReferencingDocumentInfo) : DeleteRes
     /// <summary>
     /// A transient failure due to a transaction write conflict
     /// </summary>
-    /// <param name="FailureMessage">A message providing failure information</param>
-    public record DeleteFailureWriteConflict(string FailureMessage) : DeleteResult;
+    public record DeleteFailureWriteConflict() : DeleteResult;
```

Comment:
Removed message because it should be entirely core-provided on a write conflict.